pkg/server/server.go

```diff
@@ -516,28 +526,25 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 			klog.V(5).Infoln("DATA sent to Backend")
 
 		default:
-			klog.V(5).InfoS("Ignore packet coming from frontend",
+			klog.V(2).InfoS("Unrecognized packet from frontend",
```

Comment:
At some point we are going to want to extend our protocol. Since we cannot update all agents and servers simultaneously this will happen. Not sure we want this as V2 and littering our logs?

Reply:
Yeah, that makes sense. This seems like a good usecase for the packet counts metric...